charts/netdata/values.yaml

```diff
@@ -28,19 +28,16 @@ sd:
         cpu: 50m
         memory: 100Mi
 
-sysctlImage:
-  enabled: false
+initContainerImage:
```

Comment:
Nit: if we are using this image for all init containers, then maybe we should name this key with a plural form of _container_ word, like _initContainersImage_ ?

Reply:
Fixed